application/apps/indexer/session/src/handlers/observing/mod.rs

```diff
@@ -162,6 +164,14 @@ async fn run_producer<T: LogMessage, P: Parser<T>, S: ByteSource>(
                         }
                         MessageStreamItem::Done => {
                             trace!("observe, message stream is done");
+                            let elapsed = timer.elapsed();
+                            println!("---------------------------------------------------------");
+                            println!("---------------------------------------------------------");
+                            println!("---------------------------------------------------------");
+                            println!("File Read Took: {}", elapsed.as_millis());
+                            println!("---------------------------------------------------------");
+                            println!("---------------------------------------------------------");
+                            println!("---------------------------------------------------------");
```

Comment:
please remove println! s from production code

Reply:
Sorry I've forgot to drop one temporary commit. It's dropped now :+1: